assets/global.js

```diff
@@ -879,7 +892,42 @@ class SlideshowComponent extends SliderComponent {
   }
 }
 
-customElements.define('slideshow-component', SlideshowComponent);
+customElements.define('slideshow-section-component', SlideshowSectionComponent);
+
+class AnnouncementBarSlider extends SlideshowSharedComponent {
+  constructor() {
+    super();
+
+    this.announcementBarArrowButtonWasClicked = false;
+
+    [this.reducedMotion, this.desktopLayout].forEach((mediaQuery) => {
+      mediaQuery.addEventListener('change', () => {
+        if (this.slider.getAttribute('data-autoplay') === 'true') this.setAutoPlay();
+      });
+    });
+
+    [this.prevButton, this.nextButton].forEach((button) => {
```

Comment:
This can extend the `onButtonClick` function that is already attached to these elements:

```js
  onButtonClick(event) {
    super.onButtonClick(event);
    this.announcementBarArrowButtonWasClicked = true;
  }
```